peachjam/migrations/0152_backfill_natures.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.14 on 2024-08-29 13:58
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    # backfill natures for existing documents
+    DocumentNature = apps.get_model("peachjam", "DocumentNature")
+
+    DOC_TYPES = [
+        ("core_document", "document", "Document"),
+        ("gazette", "gazette", "Gazette"),
+        ("generic_document", "document", "Document"),
+        ("judgment", "judgment", "Judgment"),
+        ("legal_instrument", "legal-instrument", "Legal Instrument"),
+        ("legislation", "act", "Act"),
+        ("book", "book", "Book"),
+        ("journal", "journal", "Journal"),
+    ]
+    for doc_type, nature, name in DOC_TYPES:
+        nature = DocumentNature.objects.get_or_create(code=nature)[0]
+        # get_or_create with defaults={name: name} doesn't seem to work
+        if not nature.name:
+            nature.name = name
+            nature.save()
```

Comment:
@longhotsummer this is still not working for me :thinking:

Reply:
I wonder if it's related to the translated names functionality on those fields? Try setting `nature.name_en = name` as well?